src/docs/GetStarted/Developers/npmScriptTutorial/index.mdx

```diff
@@ -182,17 +182,16 @@ Add in your package.json the following script :
 },
 ```
 
-I know it's not the most elegant way of doing thing, but it is only for the example's sake. 😅
 
 ### B) Create a dev server :
 
-First let's install `live-server`
+First, let's install `live-server`
 
 ```shell
 npm install live-server --save-dev
 ```
 
-And create a scritp to lunch it upon our `dist/` directory. We will tell it to refresh the browser whenever there is a change in the dist directory, and also to run the `statics:copy` command before starting.
+And create a script to lunch it upon our `dist/` directory. We will tell it to refresh the browser whenever there is a change in the dist directory, and also to run the `statics:copy` command before starting.
```

Comment:
And create a script to launch it it upon our `dist/` directory. This will refresh the browser whenever these is a change in the `dist` directory and will run the `statics:copy` command before starting.

Reply:
lunch - lol